server/bigbluebuttonapiwrapper/api/APIcalls.go

```diff
@@ -0,0 +1,321 @@
+/*
+Copyright 2018 Blindside Networks
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"github.com/blindsidenetworks/mattermost-plugin-bigbluebutton/server/bigbluebuttonapiwrapper/dataStructs"
+	"github.com/blindsidenetworks/mattermost-plugin-bigbluebutton/server/bigbluebuttonapiwrapper/helpers"
+	"log"
+	"net/url"
+	"strconv"
+)
+//url of the BigBlueButton server
+var BaseUrl string
```

Comment:
Why not make these part of the `Plugin` struct? Makes it easier to test.

Reply:
`BaseUrl` is used in many functions which are not part of plugin struct.